pkg/master/master.go

```diff
@@ -420,8 +422,7 @@ func (m *Master) InstallAPIs(apiResourceConfigSource serverstorage.APIResourceCo
 		}
 		apiGroupInfo, enabled := restStorageBuilder.NewRESTStorage(apiResourceConfigSource, restOptionsGetter)
```

Comment:
`!enabled` does not currently mean an error occurred. see `pkg/registry/authorization/rest/RESTStorageProvider#NewRESTStorage` as an example. if we want to turn this into a fatal error, we actually need to plumb an error response from NewRESTStorage